migrations/schema.txt

```diff
@@ -117,6 +117,7 @@ pod_versions
 | created_at | 1114 | "timestamp without time zone" | nil                                        | true       | false       | :datetime | nil          |
 | updated_at | 1114 | "timestamp without time zone" | nil                                        | true       | false       | :datetime | nil          |
 | pod_id     | 23   | "integer"                     | nil                                        | false      | false       | :integer  | nil          |
+| deleted    | 16   | "boolean"                     | "false"                                    | true       | false       | :boolean  | false        |
```

Comment:
Shouldn't this have an `is_` prefix like the other boolean fields?

- [`is_active`](https://github.com/CocoaPods/Humus/blob/master/migrations/schema.txt#L199)
- [`is_verified`](https://github.com/CocoaPods/Humus/blob/master/migrations/schema.txt#L100)